content/src/service/garbage-collection/GarbageCollectionManager.ts

```diff
@@ -78,7 +77,7 @@ export class GarbageCollectionManager {
   private waitUntilSyncFinishes(): Promise<void> {
     return new Promise(async (resolve) => {
       while (this.sweeping === true) {
-        await delay('1s')
+        await setTimeout(() => {}, 1000)
```

Comment:
not quite sure if this is right, it seems the classic timeout, not the one that returns the promise